packages/gestalt/src/TableOfContents.css

```diff
@@ -0,0 +1,7 @@
+.wrapper {
+  border-left: 1px solid var(--color-border-container);
+}
+
+.ul {
```

Comment:
Nit: It feels a little odd to see gap without flex nearby. Take a look at the composes pattern, but you can also do something  like

```
.TOCWrapper{
  composes Flex from "./Layout.css"
  gap:4
}
```